rules/windows/malware/av_webshell.yml

```diff
@@ -14,14 +14,14 @@ logsource:
     product: antivirus
 detection:
     selection:
-        Signature:
-            - "PHP/Backdoor*"
-            - "JSP/Backdoor*"
-            - "ASP/Backdoor*"
-            - "Backdoor.PHP*"
-            - "Backdoor.JSP*"
-            - "Backdoor.ASP*"
-            - "*Webshell*"
+        Signature|startswith:
+            - "PHP/Backdoor"
+            - "JSP/Backdoor"
+            - "ASP/Backdoor"
+            - "Backdoor.PHP"
+            - "Backdoor.JSP"
+            - "Backdoor.ASP"
+            - "*Webshell"
```

Comment:
also, `Webshell` had `contains` logic in original rule, and this one with `*` will not work.
you can do the same hack as I proposed previously:

```
    selection:
        - Signature|startswith:
        - Signature|contains:
```

Reply:
The converted rule will have the contains effect
`winlog.event_data.Signature.keyword:(PHP\/Backdoor* OR JSP\/Backdoor* OR ASP\/Backdoor* OR Backdoor.PHP* OR Backdoor.JSP* OR Backdoor.ASP* OR *Webshell*)`
But i'm going to fix best practice

Reply:
Also, i dont think this hack is going to work there, this will produce a AND condition, not OR

Reply:
Hm, let me check this out. Here is the rule:

```
title: Antivirus Web Shell Detection
id: fdf135a2-9241-4f96-a114-bb404948f736
description: Detects a highly relevant Antivirus alert that reports a web shell
date: 2018/09/09
modified: 2019/10/04
author: Florian Roth
references:
    - https://www.nextron-systems.com/2018/09/08/antivirus-event-analysis-cheat-sheet-v1-4/
tags:
    - attack.persistence
    - attack.t1100
    - attack.t1505.003
logsource:
    product: antivirus
detection:
    selection:
        - Signature|startswith:
            - "PHP/Backdoor"
            - "JSP/Backdoor"
            - "ASP/Backdoor"
            - "Backdoor.PHP"
            - "Backdoor.JSP"
            - "Backdoor.ASP"
        - Signature|contains:
            - "Webshell"
    condition: selection
fields:
    - FileName
    - User
falsepositives:
    - Unlikely
level: critical
```

Here is the result of conversion: 

```
$ tools/sigmac -t es-qs -c tools/config/elk-windows.yml /tmp/rule.yml
(Signature.keyword:(PHP\/Backdoor* OR JSP\/Backdoor* OR ASP\/Backdoor* OR Backdoor.PHP* OR Backdoor.JSP* OR Backdoor.ASP*) OR Signature.keyword:(*Webshell*))
```

Seems fine to me %)

Reply:
Sorry, I was missing the "-" hehe

Reply:
Fixed :)